### PR TITLE
First PR of the pure python Pypixelbuf library

### PR DESCRIPTION
**Pull request by rhooper**
This is a preliminary PR for pypixelbuf @kattni 
There are some changes needed to shrink the code still.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - "3.6"
+
+cache:
+  pip: true
+
+# TODO: if deployment to PyPi is desired, change 'DEPLOY_PYPI' to "true",
+#       or remove the env block entirely and remove the condition in the
+#       deploy block.
+env:
+  - DEPLOY_PYPI="false"
```
Comment (tannewt) [inline .travis.yml]: I think you will want to do this.

File: pypixelbuf.py [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+class LGRB(ByteOrder):
+    byteorder = (2, 1, 3, 0)
+    bpp = 4
+    has_luminosity = True
+
+
+class LGBR(ByteOrder):
+    byteorder = (2, 3, 1, 0)
+    bpp = 4
+    has_luminosity = True
```
Comment (tannewt) [inline pypixelbuf.py]: Weren't we going to replace these classes with strings like "RGB"?

**Review by tannewt — CHANGES_REQUESTED** [review]
Thanks for this repo! Do you mean to follow up with the API changes?

File: pypixelbuf.py (hunk 2 above)
Comment (rhooper) [reply pypixelbuf.py]: We are.  This was to get the PR in place to start moving forward.

Comment (tannewt) [reply pypixelbuf.py]: Ok, sounds good!

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      tags: true
+      condition: $DEPLOY_PYPI = "true"
+
+install:
+  - pip install -r requirements.txt
+  - pip install circuitpython-build-tools Sphinx sphinx-rtd-theme
+  - pip install --force-reinstall pylint==1.9.2
+
+script:
+  - pylint adafruit_dotstar.py
```
Comment (tannewt) [inline .travis.yml]: The build is failing because this is incorrect.

**Review by tannewt — APPROVED** [review]
This is a great start! Thank you!

**Pull request merged**